site/.gitignore

```diff
@@ -1,2 +1,4 @@
 /data
 /target
+frontend/node_modules
+frontend/dist/**/
```

Comment:
you'll want to add those to dockerignore too, otherwise it'll send them to the build service when doing that locally.